doc/can-query-logic.md

```diff
@@ -14,139 +14,141 @@ data caching and real-time behavior.
-}
-```
+  },[
+    {id: 1, name: "do dishes", complete: false},
+    {id: 2, name: "mow lawn", complete: true},
+    // ...
+  ]);
+  console.log( filter ); //-> [{id: 1, name: "do dishes", complete: false}]
 
-@param {function|can-reflect/schema} schemaOrType Defines the behavior of
-keys on a [can-query-logic/query]. This is done with either:
-
-  - A constructor function that supports [can-reflect.getSchema can-reflect.getSchema]. Currently, [can-define/map/map] supports the `can.getSchema` symbol:
-    ```js
-    import {DefineMap} from "can";
-
-    const Todo = DefineMap.extend({
-        id: {
-            identity: true,
-            type: "number"
-        },
-        name: "string",
-        complete: "boolean"
-    });
-    new QueryLogic(Todo);
-    ```
-  - A [can-reflect.getSchema schema object] that looks like the following:
-    ```js
-    import {MaybeNumber, MaybeString, MaybeBoolean} from "can"
-    new QueryLogic({
-        // keys that uniquely represent this type
-        identity: ["id"],
-        keys: {
-            id: MaybeNumber,
-            name: MaybeString,
-            complete: MaybeBoolean
-        }
-    })
-    ```
-
-    Note that if a key type (ex: `name: MaybeString`) is __not__ provided, filtering by that
-    key will still work, but there won't be any type coercion. For example, the following
-    might not produce the desired results:
-
-    ```js
-    var queryLogic = new QueryLogic({identity: ["id"]});
-    queryLogic.union(
-        {filter: {age: 7}},
-        {filter: {age: "07"}}) //-> {filter: {age: {$in: [7,"07"]}}}
-    ```
-    Use types like [can-data-types/maybe-number/maybe-number] if you want to add basic
-    type coercion:
-
-    ```js
-    var queryLogic = new QueryLogic({
-        identity: ["id"],
-        keys: {age: MaybeNumber}
-    });
-    queryLogic.union(
-        {filter: {age: 7}},
-        {filter: {age: "07"}}) //-> {filter: {age: 7}}
-    ```
-
-    If you need even more special key behavior, read [defining properties with special logic](#Definingfilterpropertieswithspeciallogic).
+  ```
+  @codepen
+  @highlight 6-15,only
+
+@param {function|can-reflect/schema} schemaOrType Defines the behavior of keys on a [can-query-logic/query]. This is done with either:
+
+- A constructor function that supports [can-reflect.getSchema can-reflect.getSchema]. Currently, [can-define/map/map] supports the `can.getSchema` symbol:
+
+  @sourceref ./can-query-logic-models/todo-example.js
+  @codepen
+  @highlight 3,10,only
+
+- A [can-reflect.getSchema schema object] that looks like the following:
+
+  ```js
+  import {QueryLogic, MaybeNumber, MaybeString, MaybeBoolean} from "can";
+
+  const queryLogic = new QueryLogic({
+    // keys that uniquely represent this type
+    identity: ["id"],
+    keys: {
+      id: MaybeNumber,
+      name: MaybeString,
+      complete: MaybeBoolean
+    }
+  });
+
+  const result = queryLogic.filterMembers({ filter: {complete: false}}, [
```

Comment:
I would not show filterMembers here ... that's extra that doesn't really matter

Reply:
What would you show, `canReflect.getSchema`?